python/cudf_polars/cudf_polars/containers/dataframe.py

```diff
@@ -141,7 +141,9 @@ def from_table(cls, table: plc.Table, names: Sequence[str]) -> Self:
         if table.num_columns() != len(names):
             raise ValueError("Mismatching name and table length.")
         return cls(
-            Column(c, name=name) for c, name in zip(table.columns(), names, strict=True)
+            # TODO: Pass along dtypes here
```

Comment:
Just confirming: this is a future TODO and not for this PR?

Reply:
Yes correct.

(For context this spot could be tricky because this API doesn't, yet, provide context about what the Polars types could be from the `pylibcudf.Table`, especially for nested types)